content/6-network-security/4-fully-private-cluster/1-create-cluster/index.en.md

```diff
@@ -307,35 +329,33 @@ This Workshop would additionally require these VPC endpoints additionally. We wo
 
 Get the Security group used by the existing VPC Endpoints.
 
-
 ```bash
-export VPCE_SG=$(aws ec2 describe-vpc-endpoints --filter Name=vpc-id,Values=$CLUSTER_VPC Name=service-name,Values=*ec2 --query VpcEndpoints[].Groups[].GroupId --output text)
+export VPCE_SG=$(aws ec2 describe-vpc-endpoints --filter Name=vpc-id,Values=$PRIVATE_CLUSTER_VPC Name=service-name,Values=*ec2 --query VpcEndpoints[].Groups[].GroupId --output text)
 ```
 
-
-Create The VPC endpoints.
-
 VPC endpoint for ELB
 
 ```bash
 aws ec2 create-vpc-endpoint \
-    --vpc-id $CLUSTER_VPC \
+    --vpc-id $PRIVATE_CLUSTER_VPC \
     --vpc-endpoint-type Interface \
     --service-name com.amazonaws.$AWS_REGION.elasticloadbalancing \
-    --subnet-ids $AZ1_SUBNET $AZ2_SUBNET \
+    --subnet-ids $PRIVATE_AZ1_SUBNET $PRIVATE_AZ2_SUBNET $PRIVATE_AZ3_SUBNET \
     --security-group-id $VPCE_SG \
     --tag-specifications 'ResourceType=vpc-endpoint,Tags=[{Key=service,Value=ELB}]'
 ```
 
 Check sample output. This will is the JSON representation of the resource created.
+
 ::::expand{header="Check Output"}
-```
+
+```bash
```

Comment:
Remove bash we don’t want user to copy this example output

Reply:
done